rest/taskana-rest-spring-example/src/main/resources/application.properties

```diff
@@ -26,5 +26,7 @@ taskana.ldap.groupSearchFilterValue=groupOfUniqueNames
 taskana.ldap.groupNameAttribute=cn
 taskana.ldap.minSearchForLength=3
 taskana.ldap.maxNumberOfReturnedAccessIds=50
+####### JobScheduler delay between runs in milliseconds
```

Comment:
This is not true anymore, is it?
BTW: what is the minimum time between two starts when we are using cron? Is it possible to start every second if we would like to?

Reply:
Ok, I overlooked the comment. A cron expression for Spring has 6 parts:
<second> <minute> <hour> <day-of-month> <month> <day-of-week>
we set <second> to 0 and all other to '*'. This means, the expression triggers each full minute.
If you set also second to '*', the scheduler would run every second.

Reply:
Hmm, in the previous append some content has been lost...
The correct explanation should be:
A cron expression for Spring has 6 parts:
second minute hour day-of-month month day-of-week
when we specify  0 * * * * *, we set second to 0 and all other to asterisk. This means, the expression triggers each full minute.
If you set also second to asterisk, the scheduler would run each second.